backoff_tests.py

```diff
@@ -214,7 +337,7 @@ def test_on_exception_giveup():
                           on_backoff=log_backoff,
                           on_giveup=log_giveup,
                           max_tries=3,
-                          jitter=lambda: 0,
+                          jitter=backoff.random_jitter,
```

Comment:
jitter should be disabled here with `jitter=None` if I'm not missing something...